src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-762.mdx

```diff
@@ -0,0 +1,23 @@
+---
+subject: Android agent
+releaseDate: '2024-12-06'
+version: 7.6.2
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.6.2.zip'
+---
+
+## New Features
+
+- Implemented OpenTelemetry (Otel) Distributed Tracing.
+- Added support for Gzip compression for log data ingestion.
+- Included ANR (Application Not Responding) thread data in the reporting process.
+
+## Bug Fixes
+
+- Resolved an issue with log instrumentation that caused build failures.
+- Fixed a problem where throwables generated repetitive logs in logcat.
+- Addressed a crash caused by calling `removeAttributes` before the agent was started.
+- Corrected various user-reported issues related to log functionality.
+
+## Support Statement
+
+We recommend updating the agent at least every three months. Specific policies and timelines regarding the support of Android agents can be found in the [Mobile Monitoring Agents EOL Policy](/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-agents-eol-policy/#android-eol).
```

Comment:
We recommend updating the agent at least every three months. The specific policies and dates for Android agent support can be found in the [Mobile Monitoring Agents EOL Policy](/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-agents-eol-policy/#android-eol).

Reply:
looks good